snapshotter/devmapper/devmapper_test.go

```diff
@@ -56,13 +58,26 @@ func TestSnapshotterSuite(t *testing.T) {
 			return nil, nil, err
 		}
 
+		removePool := func() error {
+			if err := snap.pool.RemovePool(ctx); err != nil {
```

Comment:
No. In general we don’t want to delete pool when snapshotter stops. This is test specific resource cleaning.

Reply:
Can you add a comment that we're deleting it between test runs?

Reply:
We don’t delete pool between. Test are running in parralel, so each test has it’s own devmapper pool. It’s rather cleaning after test is done.

Reply:
Yes, but can you add a comment that the pool is a temporary test artifact and you're deleting it because it's bound to the test instead of being persistent?